src/web-ui/xdn/browser.js

```diff
@@ -0,0 +1,11 @@
+import installDevtools from "@xdn/devtools/install";
```

Comment:
Add license header and documentation explaining what this code is doing.

Reply:
Added